charts/jwt-to-rbac/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 0.6.6
+appVersion: 0.6.7
```

Comment:
Please let the appVersion 0.6.6 because only the chart was changed.

Reply:
Thanks. I made the change!